docs/source/performance/caching.mdx

```diff
@@ -0,0 +1,225 @@
+
+If a subgraph does not specify a `max-age`, the gateway will assume its response (and in turn, the overall response) cannot be cached. To override this behavior, you can set the `cache-control` header in the `didReceiveResponse` method of a `RemoteGraphQLDataSource`.
+
+Additionally, if the gateway should ignore `cache-control` response headers from subgraphs that will affect the operation's cache policy, then you can set the `honorSubgraphCacheControlHeader` property of a `RemoteGraphQLDataSource` to `false` (this value is `true` by default).
+
+The effect of setting `honorSubgraphCacheControlHeader` to `false` is to have no impact on the cacheability of the response in either direction. In other words, this property won’t determine whether the response can be cached, but it does exclude a subgraph's `cache-control` header from consideration in the gateway's calculation. If all subgraphs are excluded from consideration when calculating the overall `cache-control` header, the response sent to the client will not be cached.
+
+There are detailed examples below that illustrate the resulting behavior when overriding this header for a subgraph.
+
+## Example maxAge calculations
+
+### Entity extensions
+
+Consider the following subgraph schemas:
+
+```graphql:title=Astronauts%20Subgraph
+type Astronaut @key(fields: "id") @cacheControl(maxAge: 20) {
+  id: ID!
+  name: String
+}
+
+type Query {
+  astronaut(id: ID!): Astronaut
+  astronauts: [Astronaut]
+}
+```
+
+```graphql:title=Missions%20Subgraph
+type Mission {
+  id: ID!
+  crew: [Astronaut]
+  designation: String!
+  startDate: String
+  endDate: String
+}
+
+extend type Astronaut @key(fields: "id") {
+  id: ID! @external
+  missions: [Mission]
+}
+
+type Query {
+  mission(id: ID!): Mission @cacheControl(maxAge: 10)
+  missions: [Mission]
+}
+```
+
+For the following query:
+
+```graphql
+query GetMissionWithCrew {
+  mission(id: 1) {
+    designation
+    crew {
+      name
+    }
+  }
+}
+```
+
+The response will be considered **not cacheable**.
+
+The `cache-control` header in the response from the astronauts subgraph will be `max-age=20, public` based on the cache hint applied to the `Astronaut` type, which will be taken into consideration when this subgraph responds to the `_entities` query from the gateway.
+
+The missions subgraph, however, will not include a `cache-control` header in its response to the gateway, thus indicating that it is not cacheable, and in turn, making the entire response sent to the client not cacheable. At first glance, this behavior may seem unexpected because the `@cacheControl(maxAge: 10)` directive is applied to the `mission` root query field. But upon closer inspection, we see that this field returns the `Mission` type which contains a non-scalar `Astronaut` field and this type will have a [default `maxAge` of `0` applied](https://www.apollographql.com/docs/apollo-server/performance/caching/#default-maxage).
```

Comment:
"may seem" -> "might seem"

Reply:
"will not include" -> "does not include"

"thus indicating that it is not cacheable, and in turn" -> "thus indicating that it is not cacheable. Consequently, the entire response sent to the client is not cacheable."